### DISCUSSION TITLE
Authentik

### DISCUSSION THREAD

**Discussion by rkcreation** [coollabsio/coolify#1663] [Feature Requests]
Authentik is an open-source Identity Provider
https://goauthentik.io/

**Comment by IRediTOTO** [comment]
I just try to install your package, but look like it not working. It return page not found, all services has status green. I tried to open port 9000 but it still not work. Can you take a look?

**Reply by nizetic** [reply]
Hey. I finally figured it out. For me it was the custom domain issue. Forgot to include the port as part of the domain.
It should be like this:
![image](https://github.com/user-attachments/assets/4c629c1a-0074-4ad8-8061-55f034ef926d)

Make sure to save changes and restart the app, after a few minutes Authentik should open on `example.com`
